apps/studio/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "dev": "next dev --turbopack -p 8082",
-    "build": "next build && ./../../scripts/upload-static-assets.sh",
+    "build": "next build && if [ \"$SKIP_ASSET_UPLOAD\" != \"1\" ]; then ./../../scripts/upload-static-assets.sh; fi",
```

Comment:
@ivasilov Added this to skip the asset upload, rather than a new script. Thoughts?